report_export.py

```diff
@@ -17,7 +17,8 @@
 
 fw = flywheel.Client(fw_api_token)
 db = create_engine(db_url)
-table = "flywheel_export"
+table = "report_export"
```

Comment:
the table name should indicate the source as Flywheel (to differentiate it from other tables in the warehouse) - maybe something like `flywheel_report_export` and changing other tables to `flywheel_mri_export` etc.

Reply:
Changed the names to :
flywheel_mri_export
flywheel_report_export
flywheel_slide_export